src/services/expenses.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+const { client } = require('../utils/db');
+
+function normalize({
+  id,
+  user_id: userId,
+  spent_at: spentAt,
+  title,
+  amount,
+  category,
+  note,
+}) {
+  return {
+    id,
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
+}
+
+async function getMany(userId, category, from, to) {
+  const queryArray = [];
+  const parametersArray = [...arguments].filter(parameter => parameter).length
+    ? [...arguments]
+    : [];
+
+  queryArray.push(userId ? 'user_id=$1' : '');
+  queryArray.push(category ? 'category=$2' : '');
+  queryArray.push(from ? 'spent_at>$3' : '');
+  queryArray.push(to ? 'spent_at<$4' : '');
```

Comment:
What do you think, could you create some `enum` for this strings?

Reply:
Yes, when we use TS, I think it's a perfect case for use `enum` ))

Reply:
Thanks a lot for your reviews 😉